### PR TITLE
Changes intersection detection from intersectionRatio to isIntersecting.

### LINKED ISSUE #56: Image not loaded in random cases on Safari.
Hi, 
I am experiencing an issue where is some random cases image is not loaded in Safari. 
yall.js 3.1.1
Safari 12.1.1

Here's a quick screen recording of the issue:
https://drive.google.com/file/d/1X_tKFQF4373sqryXPo6gfT1i_gUna8C5/view

When I scroll back to that specific image which is not loaded it then gets loaded. 
Also this is not happening for a specific image, but for a random image on every page refresh.  

```
document.addEventListener("DOMContentLoaded", function () {
        yall({
            threshold: 1800
        });
 });
```

Is anyone having the same issue?

### PR DESCRIPTION
**Pull request by radfahrer**
resolves #56 by using isIntersecting rather than intersectionRatio which could be `0.0`

### CODE REVIEW

**Comment by malchata** [comment]
Is there a way we can solve #56 without introducing a new option? I'd like to resolve this bug without expanding the API if possible. [I asked in this comment in the issue](https://github.com/malchata/yall.js/issues/56#issuecomment-509769042) if it might make sense to replace `entry.intersectionRatio` with `entry.isIntersecting`, which appeared to resolve the issue—at least for me. Let me know when you think, @radfahrer.

**Comment by radfahrer** [comment]
@malchata I was just looking at that, and I expect it might work. I could imagine a scenario where `intersectionRatio == 0.0` which would evaluate as falsey. Was there a reason you went with `intersectionRatio` in the first place?

I'll test it out and let you know if that works for us.

**Comment by malchata** [comment]
The reason was to try and broaden compatibility with Edge 15's
IntersectionObserver implementation without polyfills, which supports the
intersectionRatio property, but not the isIntersecting property. In
retrospect, this might not have been the best choice, but it's what it is.

If you can verify that switching to the isIntersecting property fixes the
issue, let me know and I can provision a fix and release it to npm.

File: src/yall.mjs [hunk 1]
```diff
@@ -89,7 +88,7 @@ export default function (options) {
   if (io in win && `${io}Entry` in win) {
```
[the remark this answers is not in the document]
Comment (radfahrer) [reply src/yall.mjs]: I was able to confirm and made the update you suggested.

**Comment by radfahrer** [comment]
@malchata This should be ready to merge. Let me know if you want any other changes.

**Pull request merged**